src/app_engine/constants.py

```diff
@@ -74,6 +74,9 @@
 PARAM_MESSAGE = 'message'
 PARAM_ROOM_ID = 'roomId'
 PARAM_METADATA = 'metadata'
+PARAM_TYPE = 'type'
+
+TYPE_RINGING = 'ringing'
```

Comment:
Makes sense, especially if we add more ack messages. Changed to 'INVITE' to match the gcm message.